src/main/java/geekbrains/java2/ht2/Board.java

```diff
@@ -0,0 +1,108 @@
+package geekbrains.java2.ht2;
+
+import java.util.Arrays;
+import java.util.Scanner;
+
+class Board {
```

Comment:
\[Checkstyle\] ERROR: Utility classes should not have a public or default constructor\.